op-bindings/etherscan-client/etherscan_client.go

```diff
@@ -0,0 +1,247 @@
+package etherscanclient
```

Comment:
nit: can you rename the directory + package name to be `etherscan` and then its `etherscan.Client`